### PR TITLE
Tami4: attempt on adding missing docs

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

- attempt to document some undocumented features. 
- However, descriptions are based on information I could find on the web and might not be correct
- Help needed
- related to <https://github.com/home-assistant/home-assistant.io/pull/34421>

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/124506, 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced a new "Button" category for Tami4 Edge integration, allowing users to control devices.
	- Added buttons for "Boil water" and "Prepare drink."
	- New sensors provide information on UV lamp and filter statuses.
- **Documentation**
	- Expanded descriptions to clarify integration capabilities.
	- Added "Prerequisites" section outlining requirements for use.
	- Introduced "Entities" section detailing available buttons and sensors.

### CODE REVIEW

**Comment by c0ffeeca7** [comment]
@Guy293 , I tried adding some docs to this. However, I'm not entirely sure how accurate the descriptions are. Would you be willing to review and improve the content, where needed?

File: source/_integrations/tami4.markdown [hunk 1]
```diff
@@ -15,11 +16,35 @@ ha_platforms:
...
 {% include integrations/config_flow.md %}
+
+## Entities
+
+The Tami4 Edge integration provides the following entities.
+
+### Buttons
+
+- **Boil water**: Select this button to start boiling the water.
+- **Prepare drink**: Select this button to start preparing the drink.
```
Comment (joostlek) [inline source/_integrations/tami4.markdown]: I think there's a button now for every drink that's added, not just a single one

**Review by joostlek — COMMENTED** [review]
@Guy293 can you also take a look?

**Review by frenck — APPROVED** [review]
Thanks, @c0ffeeca7 👍 

../Frenck

**Pull request merged**